microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -31,6 +33,30 @@
     """,
 )
 class InfinibandSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. List all available network interfaces
+        2. Check if InfiniBand cards are present
+        3. Ensure InfiniBand cards are named starting with "ib0"
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_features=[Infiniband]),
+    )
+    def verify_ib_naming(self, log: Logger, node: Node) -> None:
+        ib_interfaces = node.features[Infiniband].get_ib_interfaces()
+        ib_first_device_name = ib_interfaces[0].ib_device_name
```

Comment:
Your test descriptions says that the test checks all infiniband cards have name starting with "ib0" whereas in code you have only checked first device. Can you please clarify this?

Reply:
Modified the test description for clarification.